### DISCUSSION TITLE
Stand up Wednesday May 3rd

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [Bugs-io/TC3001B#75] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on graphics
 - Got familiar with NextJS
 - Tried to integrate frontend and backend development branches

**What are your priorities for the next 24 hours?**:
 - Integrate frontend and backend
 - Work on retrieval of internal dataset information

**Blockers**:
 - Time

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Started working on cloud sql connection
 
**What are your priorities for the next 24 hours?**:
 - Finished db connection
 
**Blockers**:
 - Don´t know why connection is been refused by google
 
**Shoutouts**:
 - @Murcia University

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Added navbar
 - Added logout
 - Started working on file uploading styling

**What are your priorities for the next 24 hours?**:
 - Finished file uploading component and hopefully successfully conect it with back

**Blockers**:
 - None
 
**Shoutouts**:
 - @Tecdemonterrey for such good classes
